### PR TITLE
Document keystone constructor config options

### PR DESCRIPTION
**Pull request by Vultraz**
I wanted to document the `Keystone` constructor's config options. Is there a good reference for the exact purpose of most of these?

### CODE REVIEW

File: packages/keystone/README.md [hunk 1]
```diff
@@ -6,14 +6,41 @@ order: 1
...
+});
+```
+
+### Config
+
+| Option                  | Type       | Default    | Description                                                                                                                                                 |
+| ----------------------- | ---------- | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `defaultAccess`         | `Object`   | `{}`       | FOO                                                                                                                                                         |
+| `adapters`              | `Array`    | `[]`       | FOO                                                                                                                                                         |
+| `adapter`               | `Object`   | `null`     | Required. The database storage adapter. See the [Adapter Framework](https://v5.keystonejs.com/keystone-alpha/keystone/lib/adapters/) page for more details. |
```
Comment (MadeByMike) [inline packages/keystone/README.md]: `null` is not really the default so I suggest Moving  required into the default col similar to other docs:

```suggestion
| `adapter`               | `Object`   |  Required     | The database storage adapter. See the [Adapter Framework](https://v5.keystonejs.com/keystone-alpha/keystone/lib/adapters/) page for more details. |
```

File: packages/keystone/README.md [hunk 2]
```diff
@@ -6,14 +6,41 @@ order: 1
...
+
+```javascript
+const keystone = new Keystone({
+  /*...config */
+});
+```
+
+### Config
+
+| Option                  | Type       | Default    | Description                                                                                                                                                 |
```
Comment (MadeByMike) [inline packages/keystone/README.md]: Can we remove `FOO` from all of these? I'll try and help with some descriptions today if I get time :)

**Comment by MadeByMike** [comment]
Thanks so much for this!

**Comment by Vultraz** [comment]
Dropped the `FOO`s, added a `require` line to the JS example, reordered the list a bit to group related options, and clarified the `secureCookies` default.

**Comment by MadeByMike** [comment]
Looks great! I'm prepared to merge this and come back and add more detail later :)

**Pull request merged**